src/service/navigation.js

```diff
@@ -435,6 +440,36 @@ export class Navigation {
     return false;
   }
 
+  /**
+   * Handles clicking on links with possible redirects.
+   * @param {!Event} e
+   * @param {!Element} target
+   * @return {boolean}
+   * @private
+   */
+  handleRedirect_(e, target) {
+    const safeRedirectAttribute = 'data-saferedirectreason';
+    const safeRedirectLink = target.getAttribute('data-saferedirecturl');
+    const isUnsafeRedirect = target.hasAttribute(safeRedirectAttribute);
```

Comment:
I don't understand the naming of these two attributes.

Reply:
`data-saferedirectreason` seems to imply that the link was deemed unsafe. Confusing name though.